A2crd new chord ext

weltyj opened the pull request.

sciurius commented:
I have reworked some of the A2Crd stuff and added your changes. Please checkout the latest 'dev' branch.

You can close (and remove) your `a2crd_new_chord_ext` branch.

weltyj commented:
Thanks, I'll grab the new dev branch.






On Wednesday, October 28, 2020, 4:09:05 AM PDT, Johan Vromans <notifications@github.com> wrote: 







I have reworked some of the A2Crd stuff and added your changes. Please checkout the latest 'dev' branch.

You can close (and remove) your a2crd_new_chord_ext branch.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.

sciurius commented:
The new test is `t/70_a2crd.t`. It processes all `.crd` files in the `t/a2crd` directory and compares the result to a corresponding `.cho` file in the `t/a2crd` directory. If the compare fails, it leaves a `.tmp` file for visual inspection.
So adding new test cases is just a matter of putting the `.crd` into the `t/a2crd` directory and adding a `.cho` with the desired result.
